tjenestebuss-integrasjon/src/main/kotlin/no/nav/pensjon/brev/tjenestebuss/tjenestebussintegrasjon/Routing.kt

```diff
@@ -91,6 +93,8 @@ fun Application.tjenestebussIntegrationApi(config: Config) {
         val samhandlerTjenestebussService = SamhandlerTjenestebussService(config.getConfig("services.tjenestebuss"), stsSercuritySOAPHandler)
         val arkivTjenestebussService = ArkivTjenestebussService(config.getConfig("services.tjenestebuss"), stsSercuritySOAPHandler)
         val dokumentProduksjonService = DokumentproduksjonService(config.getConfig("services.dokprod"), stsSercuritySOAPHandler)
+        val extreamBrevService = ExtreamBrevService(config.getConfig(""), stsSercuritySOAPHandler)
```

Comment:
Skulle det vært noe i tomme quotes her?

Reply:
bra catch, det er faktisk by design, jeg henter config fra rootnivå og deretter benyttes den i to forskjellige grener senere. men jeg hadde faktisk skrevet feil path for tjenestebuss url :)
    private val tjenestebussUrl = config.getString("tjenestebuss.url")

    systemId = config.getString("brevklient.systemid")
    passord = config.getString("brevklient.passord")
    rootURL = config.getString("brevklient.rooturl")